Source/CarthageKit/Project.swift

```diff
@@ -762,6 +808,37 @@ public final class Project {
 					}
 			}
 	}
+	
+	/// Checks the version files for the dependency and each of the dependencies
+	/// listed in its Cartfile.
+	///
+	/// Returns true if the dependency does not need to be rebuilt.
+	private func shouldSkipBuildForDependency(_ dependency: Dependency<PinnedVersion>, dependencyProjects: Set<ProjectIdentifier>, dependenciesToBeBuilt: [Dependency<PinnedVersion>], platforms: Set<Platform>) -> SignalProducer<Bool, CarthageError> {
```

Comment:
This could use a Swift 3 name.

```swift
private func shouldSkipBuild(
    for dependency: Dependency<PinnedVersion>,
    dependingOn dependencyProjects: Set<ProjectIdentifier>,
    whenBuilding dependenciesToBeBuilt: [Dependency<PinnedVersion>],
    on platforms: Set<Platform>
) -> SignalProducer<Bool, CarthageError> {